notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1427 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        service_account=SERVICE_ACCOUNT,
</pre><p>Gets me an error for service account. <code>run()</code>method does not have service_account param. It's unknown! Please remove it.</p><p> I removed and it worked. So wondering if this worked for you?!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>You are right. </p><p>My apologies. This was supposed to be added for the Pipeline Job but not here. </p><p>Fixed it in the latest commit.</p>

Reply:
<p>No problem! Thank you!</p>